packages/playground/website/src/components/terminal-button/index.tsx

```diff
@@ -0,0 +1,273 @@
+          await playground?.writeFile('/tmp/stderror', '');
+          await playground?.writeFile('/wordpress/run-cli.php',
+            `<?php
+            $GLOBALS['argv'] = [
+              "/wordpress/wp-cli.phar",
+              "--path=/wordpress/build",
+              ${wpCliArgs.join('\n')}
+            ];
+
+            function iconv_substr($str, $from, $to = 1) { return substr($str, $from, $to); }
+            function iconv_strlen($str) { return strlen($str); }
+
+            define('STDIN', fopen('php://stdin', 'rb'));
+            define('STDOUT', fopen('php://stdout', 'wb'));
+            define('STDERR', fopen('/tmp/stderr', 'wb'));
+
+            require( "/wordpress/wp-cli.phar" );
+            `
+          );
+
+          const output = await playground.run({
+            scriptPath: '/wordpress/run-cli.php'
+          });
+
+          const stderr = await playground?.readFileAsText('/tmp/stderr');
+          stderr.split('\n').forEach(line => {
+            if (line.includes('Warning')) {
+              return;
+            }
+            terminalRef.current?.writeln(line);
+          });
+
+          const decodedOutput = new TextDecoder().decode(output.body);
+          decodedOutput.split('\n').slice(1, decodedOutput.length).forEach(line => {
+            if (line.includes('Warning') || line.includes('<br />')) {
+              return;
+            }
+            console.log(line);
+            terminalRef.current?.writeln(line);
+          });
+        }
+        break;
+
+      case 'phpunit':
+        if (playground) {
+          const phpunitArgs = args.map(arg => `"${arg.replaceAll('"', '\\"')}",`)
+
+          await playground?.writeFile('/tmp/stderror', '');
+
+          await playground?.writeFile('/wordpress/run-tests.php',
+            `<?php
+            namespace {
+              $_SERVER['argv'] = [
+                '/wordpress/vendor/phpunit/phpunit/phpunit',
+                '-c',
+                'wordpress/phpunit.xml.dist',
+                ${phpunitArgs.join('\n')}
+              ];
+
+              function iconv_substr($str, $from, $to = 1) { return substr($str, $from, $to); }
+              function iconv_strlen($str) { return strlen($str); }
+
+              function proc_open( $command, $descriptor_spec, $pipes ) { return false; }
+
+              define('STDIN', fopen('php://stdin', 'rb'));
+              define('STDOUT', fopen('php://stdout', 'wb'));
+              define('STDERR', fopen('/tmp/stderr', 'wb'));
+
+              define('WP_RUN_CORE_TESTS', true);
+              putenv('WP_TESTS_SKIP_INSTALL=1');
+            }
+
+            namespace Composer {
+              require( __DIR__ . "/vendor/phpunit/phpunit/phpunit" );
+            }
+            `
+          );
+
+          try {
+            const output = await playground.run({
+              scriptPath: '/wordpress/run-tests.php'
+            });
+
+            const decodedOutput = new TextDecoder().decode(output.body);
```

Comment:
Next iteration idea: Stream the output live. This will require a function like `php.outputMode( 'file' | 'callback' )`